### ISSUE TITLE
fix(docs): In the skore cross validation example, the plotly graph is not full size in the rendering

### ISSUE THREAD

**Issue by MarieS-WiMLDS** [probabl-ai/skore#594]

**Comment by augustebaum** [comment]
I get the same behaviour in [`sphinx-gallery`'s documentation](https://sphinx-gallery.github.io/stable/auto_plotly_examples/plot_0_plotly.html) itself. It appears that this is a bug with either sphinx-gallery or plotly.

**Comment by augustebaum** [comment]
https://github.com/sphinx-gallery/sphinx-gallery/issues/1394

**Comment by augustebaum** [comment]
Related: when the Plotly figure is shown in the example, the resulting image can't be used as the thumbnail.

<details>
<summary>Detailed explanation; tldr plotly is buggy</summary>
Sphinx-gallery tells me to install orca in order to do this (see first few lines of https://sphinx-gallery.github.io/stable/auto_plotly_examples/plot_0_plotly.html), but [this issue](https://github.com/smarie/mkdocs-gallery/issues/49) in mkdocs-gallery is also relevant and is more up to date.
In fact, I tried the legacy orca method, but orca is not in PyPI so it's a no-go.

The new methd is to use plotly's kaleido, but when I tried to build the docs I got this:
https://github.com/plotly/plotly.py/issues/4832
</details>

In conclusion, using plotly images as thumbnails doesn't work yet, and the docs look like this as a result (note the big "BROKEN" image):
![Image](https://github.com/user-attachments/assets/09419e11-8103-4724-b7ab-dc60c5f89b58)

**Comment by augustebaum** [comment]
I didn't expect to hit so many roadblocks. Our options (feel free to contribute more):
- De-prioritize and wait for the issues to be solved, then try again
- Instead take a screenshot of the resulting plot and use this, hoping it will be used automatically as the thumbnail
- Try out alternatives to `sphinx-gallery`

To be completely honest, I don't think the thumbnails bring much to the docs.

@MarieS-WiMLDS @sylvaincom

**Comment by MarieS-WiMLDS** [comment]
I vote for a mix of 1 & 2: use a screenshot while this issue is fixed. And I agree that the thumbnails do not provide huge value, changing `sphinx-gallery` is way too much work compared to just have a thumbnail.

**Comment by sylvaincom** [comment]
If you just unzoom on the page it autosizes automatically
